components/gitpod-db/src/typeorm/migration/1695733993909-CodeSyncDropDeleted.ts

```diff
@@ -10,22 +10,22 @@ import { columnExists } from "./helper/helper";
 export class CodeSyncDropDeleted1695733993909 implements MigrationInterface {
     public async up(queryRunner: QueryRunner): Promise<void> {
         if (await columnExists(queryRunner, "d_b_code_sync_resource", "deleted")) {
-            await queryRunner.query("ALTER TABLE `d_b_code_sync_resource` DROP COLUMN `deleted`, ALGORITHM=INSTANT");
+            await queryRunner.query("ALTER TABLE `d_b_code_sync_resource` DROP COLUMN `deleted`");
```

Comment:
It would make sense to use `ALGORITHM=INPLACE, LOCK=NONE` here, as it's quite a lot of data. Using this is a proven pattern with v5.7 and docs on v8 still show support.